ansible/ansible.cfg

```diff
@@ -0,0 +1,4 @@
+[defaults]
+host_key_checking = False
+remote_user = root
+become_method = sudo
```

Comment:
Is `become_method` really needed here? since you were running ansible-playbook as `root`.